src/components/AccordionItem/AccordionItem.js

```diff
@@ -133,12 +135,22 @@ export default class AccordionItem extends Component {
           aria-expanded={this.state.open}
           className={`${prefix}--accordion__heading`}
           onClick={this.handleHeadingClick}>
-          <Icon
-            className={`${prefix}--accordion__arrow`}
-            icon={iconChevronRight}
-            description={iconDescription}
-            role={null} // eslint-disable-line jsx-a11y/aria-role
-          />
+          {componentsX ? (
+            <ChevronRight16
+              className={`${prefix}--accordion__arrow`}
+              icon={iconChevronRight}
+              description={iconDescription}
```

Comment:
I believe `description` should be `aria-label` here since `description` is not a valid SVG attribute (our new icon helpers map more directly to raw SVGs compared to `<Icon>`), other than that the PR looks good to me

Reply:
I've updated that now. Thanks for reviewing!